### DISCUSSION TITLE
Arbitrary values such as `min-h-[100px]` not working with imports

### DISCUSSION THREAD

**Discussion by plourenco** [tailwindlabs/tailwindcss#10132] [Help] [answered]
**What version of Tailwind CSS are you using?**

v.3.2.4

**What build tool (or framework if it abstracts the build tool) are you using?**

Next.js v13

**What version of Node.js are you using?**

v16

**Reproduction URL**

https://codesandbox.io/s/using-tailwind-with-next-js-forked-8urvz8?file=/package.json

**Describe your issue**

It seems that arbitrary values such as `min-h-[100px]` do not work in an environment where tailwind CSS utilities are imported, as opposed to generic utilities such as `min-h-0`. 

In the example I've added above, the arbitrary utility is not recognized, while a generic one targets `min-height: 0` correctly.

I could not reproduce the same issue on the tailwind playground, I suspect it might be related to Nextjs built-in PostCSS support.

Related to #6995.

**Comment by adamwathan** [comment, marked as answer]
Hey! You have no `tailwind.config.js` file and haven't configured your content sources so Tailwind doesn't know which files to scan for classes,

Check out the documentation here:

https://tailwindcss.com/docs/content-configuration

In general this project isn't configured correctly — you've got PurgeCSS set up which isn't necessary, you're using Next.js 9 when the latest version is 13, and you're using `@zeit/next-css` which isn't needed these days with recent versions of Next. I would recommend following the documentation here for setting up a Next.js + Tailwind project: https://tailwindcss.com/docs/guides/nextjs

**Reply by plourenco** [reply]
Hi. I'm actually running into this problem on a private repository and while attempting to create a reproducible example I quickly forked the first sandbox template I found with nestjs/tailwind but didn't realize this project config was incomplete or incorrect.
I just tried the same with the official template and I couldn't reproduce it, so it seems the issue may be caused by something else.

**Reply by plourenco** [reply]
It seems that the issue was actually using `classNames` in a way they couldn't be [statically extracted](https://v2.tailwindcss.com/docs/just-in-time-mode#arbitrary-value-support).
I was using string concatenation to create class names, while in fact, tailwind does not expect arbitrary values here. The best fit here is using styles instead. Thanks for pointing out the right direction.

**Comment by Mihai-github** [comment]
Hello, I have this inside my content in the tailwind.config.js file, and still not working using the Remix.js framework

```
module.exports = {
  content: [
    "./app/**/*.{js,jsx,ts,tsx}",
    "./components/**/*.{js,jsx,ts,tsx}",
    "node_modules/flowbite-react/**/*.{js,jsx,ts,tsx}",
  ],
  mode: "jit",
};
```
